packages/components/organisms/f-header/README.md

```diff
@@ -85,7 +85,7 @@ The props that can be defined are as follows:
 | userInfoUrl               | `String`      | `/api/account/details` | URL to call to retrieve the userInfo (when `userInfoProp` isn't set). |
 | showCountrySelector       | `Boolean`     | `false` | Defines whether the country selector should be shown in the navigation. |
 | showSkipLink              | `Boolean`     | `true`  | Set to false if you need to remove skip-to-main-content link from the header. |
-
+| shouldUseJetLogo                | `Boolean`     | `false`  | Set to true if you want to show Jet logo in the header. |
```

Comment:
For backwards compatibility would it be better to be default to true?

Reply:
The Jet logo will be used mostly for Ops team's projects. For customer facing sites the logos are still the same (JE amd Menulog) and their choice is covered by our globalisation service.

Reply:
cool